libqtile/widget/wttr.py

```diff
@@ -0,0 +1,88 @@
+from urllib.parse import quote, urlencode
+
+from libqtile.widget import GenPollUrl, base
+
+
+class Wttr(GenPollUrl):
+    """Display weather widget provided by wttr.in_.
+
+    .. _wttr.in: https://github.com/chubin/wttr.in/
+
+    To specify your own custom output format, use the special %-notation
+    (example: 'My_city: %t(%f), wind: %w'):
+
+        - %c    Weather condition,
+        - %C    Weather condition textual name,
+        - %h    Humidity,
+        - %t    Temperature (Actual),
+        - %f    Temperature (Feels Like),
+        - %w    Wind,
+        - %l    Location,
+        - %m    Moonphase 🌑🌒🌓🌔🌕🌖🌗🌘,
+        - %M    Moonday,
+        - %p    precipitation (mm),
+        - %P    pressure (hPa),
+        - %D    Dawn !,
+        - %S    Sunrise !,
+        - %z    Zenith !,
+        - %s    Sunset !,
+        - %d    Dusk !. (!times are shown in the local timezone)
+
+    Specify multiple locations separated with ',': ``Minsk, Reykjavik``. Cities
+    will change randomly every update.
+
+    Add the character ``~`` at the beginning to get weather for some special
+    location: ``~Vostok Station`` or ``~Eiffel Tower``.
+
+    Also can use IP-addresses (direct) or domain names (prefixed with @) to
+    specify a location:
+    ``@github.com``, ``123.456.678.123``
+    """
+
+    orientation = base.ORIENTATION_HORIZONTAL
+    defaults = [
+        ('json', False, "Don't set ``True``"),
```

Comment:
If this should never be set to `True` then can you just remove it from `defaults` entirely and hard code it into L69:
``` python
GenPollUrl.__init__(self, json=False, **config)
```
or just do:
``` python
self.json = False
```
at the end of `__init__`?

Reply:
OK. I will do it.